plugins/out_pgsql/pgsql_connections.c

```diff
@@ -54,9 +54,12 @@ void *pgsql_create_connection(struct flb_pgsql_config *ctx)
         flb_errno();
         return NULL;
     }
+
+    flb_plg_debug(ctx->ins, "connection_options: %s", ctx->conn_options);
```

Comment:
I don't think this it's actually needed, but if we will keep it please add the list of other options used as a full report of the connection string that it's being used in fluentibt

Reply:
I agree with you, I will remove the debug message